lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -476,14 +545,50 @@ defmodule ExUnit.DocTestTest do
         doctest ExUnit.DocTestTest.IndentationNotEnough
       end
     end
+  end
 
+  test "fails in fenced indentation mismatch" do
+    assert_raise ExUnit.DocTest.Error,
+      ~r[test/ex_unit/doc_test_test\.exs:\d+: indentation level mismatch: " iex> bar = 2", should have been 0 spaces], fn ->
+      defmodule NeverCompiled do
+        import ExUnit.DocTest
+        doctest ExUnit.DocTestTest.FencedIndentationMismatchedPrompt
+      end
+    end
+
+    assert_raise ExUnit.DocTest.Error,
+      ~r[test/ex_unit/doc_test_test\.exs:\d+: indentation level mismatch: "  3", should have been 0 spaces], fn ->
+      defmodule NeverCompiled do
+        import ExUnit.DocTest
+        doctest ExUnit.DocTestTest.FencedIndentationTooMuch
+      end
+    end
+
+    assert_raise ExUnit.DocTest.Error,
+      ~r[test/ex_unit/doc_test_test\.exs:\d+: indentation level mismatch: \"3\", should have been 2 spaces], fn ->
+      defmodule NeverCompiled do
+        import ExUnit.DocTest
+        doctest ExUnit.DocTestTest.FencedIndentationNotEnough
+      end
+    end
+  end
+
+  test "fails with improper termination" do
     assert_raise ExUnit.DocTest.Error,
       ~r[test/ex_unit/doc_test_test.exs:\d+: expected non-blank line to follow iex> prompt], fn ->
```

Comment:
Assuming the order our PRs are applied is uncertain, wouldn't that cause a conflict? Git's followed these lines through the move and views them as unchanged, so it should handle them... right?

Or am I confused because I mostly do rebase-merges?